src/docs/product/accounts/quotas/manage-attachments-quota.mdx

```diff
@@ -0,0 +1,118 @@
+
+<Alert level="warning">
+
+If you have an annual billing period, plan downgrades and decreases in reserved volume go into effect at the beginning of your **next billing year.**
+
+</Alert>
+
+Changes to on-demand volume typically go into effect immediately and are guaranteed to go into effect within 24 hours. However, you can't decrease your on-demand budget to less than what you've consumed in the current period.
+
+To decrease your quota, go to **Settings > Subscription** and click "Manage Subscriptions". When you reach the "Review & Confirm" step, the date that these changes go into effect will be displayed:
+
+![The Review & Confirm step of a subscription update.](review-subscription.png)
+
+<Note>
+
+We strongly recommend that you make subscription changes **before** the last day of your billing period. Depending on your time zone, in some cases, changes made on the last day of the billing period will not go into effect until the next billing period.
+
+</Note>
+
+## 2. Rate Limiting
+
+If you have enabled the storage of crash reports, you can set limits for the maximum number of crash reports that will be stored per issue. To set up these limits, go to **Settings > Security & Privacy** and make a selection in the "Store Native Crash Reports" dropdown. This setting applies to your entire organization.
+
+## 3. Inbound Data Filters
+
+SDK configuration requires changes to your source code and depends on your next deployment, but server-side filters can be easily configured per project in the "Data Filters" section of **[Project] > Settings > Inbound Filters**.
+
+While you can't set filters for attachments directly, when you set inbound filters for error events, their associated attachments are indirectly filtered. Learn more about setting data filters for error events in [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/).
+
+## 4. Size Limits
+
+Sentry imposes limits on various fields within an event, as well as the size of full events and the requests they are sent in:
+
+- Events, attachments, and requests exceeding payload size limits are immediately dropped with a `413 Payload Too Large` error. Sentry allows compressed content encoding, and applies separate limits before and after decompression.
+- Fields exceeding the individual size limits are afterwards trimmed and truncated at a best effort.
+
+To avoid using up your attachments quota — whch is based on size and not number of instaces of attachments data sent — or having the event dropped entirely, consider limiting the size of values passed into Sentry's APIs. For example, if your application attaches application state or request bodies to Sentry events, truncate them first.
+
+The precise limits may change over time. For more information, please refer to the following resources:
+
+- [Envelope Size Limits](https://develop.sentry.dev/sdk/store/#size-limits)
+- [Store Endpoint Size Limits](https://develop.sentry.dev/sdk/store/#size-limits)
```

Comment:
Are these two links intentionally the same?

Reply:
Good catch! Fixed